src/bluesky/callbacks/tiled_writer.py

```diff
@@ -599,7 +599,7 @@ def stop(self, doc: RunStop):
                     consolidator.validate(fix_errors=True)
                 except Exception as e:
                     msg = f"{type(e).__name__}: " + str(e).replace("\n", " ").replace("\r", "").strip()
-                    warn(f"Validation of StreamResource {sres_uid} failed with error: {msg}", stacklevel=2)
+                    warn(f"Validation of StreamResource {sres_uid} failed with {msg}", stacklevel=2)
```

Comment:
Would you be unhappy to add !r here in both cases?

```suggestion
                    warn(f"Validation of StreamResource {sres_uid!r} failed with {msg!r}", stacklevel=2)
```